app/dashboard/helpers.py

```diff
@@ -336,6 +341,29 @@ def process_bounty_details(bountydetails, url, contract_address, network):
             for interested in last_bounty.interested.all():
                 new_bounty.interested.add(interested)
 
+        if fments:
+            for fment in fments:
+                new_fulfillment = BountyFulfillment.objects.create(
+                    fulfiller_address=fment.get('payload', {}).get('fulfiller', {}).get('address', '0x0000000000000000000000000000000000000000'),
+                    fulfiller_email=fment.get('payload', {}).get('fulfiller', {}).get('email', ''),
+                    fulfiller_github_username=fment.get('payload', {}).get('fulfiller', {}).get('githubUsername', ''),
+                    fulfiller_name=fment.get('payload', {}).get('fulfiller', {}).get('name', ''),
+                    fulfiller_metadata=fment.get('payload', {}).get('metadata', {}),
+                    fulfillment_id=fment.get('id'),
+                    bounty=new_bounty,
+                )
+                new_fulfillment.save()
+                new_bounty.fulfillments.add(new_fulfillment)
+
+            inactive_bounties = Bounty.objects.filter(
+                github_url=url,
+                title=bountyDataPayload.get('title'),
+                current_bounty=False,
+            ).order_by('-created_on')
+
+            for inactive_bounty in inactive_bounties:
```

Comment:
We could switch this to a `value_list` on `id` of `inactivate_bounties` and do a `BountyFulfillment.objects.filter(bounty_id__in=inactive_bounty_ids).delete()` to issue the deletion once.